beacon_node/src/cli.rs

```diff
@@ -383,7 +383,7 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .requires("enable_http")
                 .value_name("ORIGIN")
                 .help("Set the value of the Access-Control-Allow-Origin response HTTP header. \
-                    Use * to allow any origin (not recommended in production). \
+                    Use * to allow any origin (not recommended in production). 
```

Comment:
I think this line was changed accidentally

```suggestion
                    Use * to allow any origin (not recommended in production). \
```